.npmrc

```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
Does this syntax actually work?

Reply:
Yes that's how you can authenticate with `npm`:
https://docs.npmjs.com/cli/v10/configuring-npm/npmrc#auth-related-configuration

`bun` implemented `publish` 3 days ago. They just call `npm` under the hood.